var/spack/repos/builtin/packages/autoconf/package.py

```diff
@@ -18,17 +18,30 @@ class Autoconf(AutotoolsPackage, GNUMirrorPackage):
             preferred=True)
     version('2.62', sha256='83aa747e6443def0ebd1882509c53f5a2133f502ddefa21b3de141c433914bdd')
     version('2.59', sha256='9cd05c73c5fcb1f5ccae53dd6cac36bb8cb9c7b3e97ffae5a7c05c72594c88d8')
-    version('2.13', sha256='f0611136bee505811e9ca11ca7ac188ef5323a8e2ef19cffd3edb3cf08fd791e')
```

Comment:
Is it no longer possible to download this? Generally we try to avoid removing old versions unless they are no longer available (even then, users could be maintaining mirrors with those old versions).

Reply:
2.13 was released 1999-01-15, autom4te, the script patched here, didn't exist yet:
```py
        filter_file('^#! /usr/bin/env perl',
                    '#! {0} -w'.format(self.spec['perl'].command.path),
                    self.prefix.bin.autom4te,
                    backup=False)
```
So since 2017, when this filter_file() was initially added, nobody could build 2.13.

BTW. That same year, Intel announced Itanium and AMD said they are working on their own 64-bit instruction set, which later became amd64/x86_64.

So 2.13 is really old, and maybe still usable, but since 2017, it wasn't buildable in spack anymore.
(and I guess nobody needed it too since then.)